src/event-mixin.js

```diff
@@ -263,10 +268,11 @@ export function removeEventListener(type, fn, optionsOrCapture) {
   if (!fn) {
     return;
   }
-  let wrapper = fn.__eventWrapper;
+  const wrappers = fn.__eventWrappers || {};
+  const wrappersForType = wrappers[type] || [];
+  const wrapper = wrappersForType.pop();
   origRemoveEventListener.call(this, type, wrapper || fn, optionsOrCapture);
   if (wrapper) {
-    fn.__eventWrapper = null;
```

Comment:
I've purposefully omitted the cleanup here, as it might be expensive to check if `fn.__eventWrappers` is actually empty. WDYT?